### PR TITLE
Improve check versions job [MAILPOET-6171]

### PR DESCRIPTION
**Pull request by costasovo**
## Description

This PR improves action that creates PRs for updating WP Images and plugin versions in tests.

### Changes

1. Simpler PR title
2. Added description with instructions for Porter
3. Add labels to the PR
4. Prevent failures if PR exists, but update it instead

## Code review notes

For updating existing PR, I chose to use force push. Alternatively, we could pull the existing branch and run all updates on to of the branch. Force push was a much simpler change. 

Here is a PR I created when testing the change with act  https://github.com/mailpoet/mailpoet/pull/5761
Note I added the label `check-versions` after the test so it is not present in the PR and I added some texts to mark the PR as testing.

## QA notes

Can be skipped.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-6171]

## After-merge notes

Ping @costasovo to update previous PRs created by the action and add labels `automated` and `check-versions`. Or update them yourself. [Search for Update used WordPress Docker](https://github.com/mailpoet/mailpoet/pulls?q=is%3Apr+Update+used+WordPress+Docker+image++).

## Tasks

- [ ] 🚫 I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] 🚫 I added sufficient test coverage
- [ ] 🚫 I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-6171]: https://mailpoet.atlassian.net/browse/MAILPOET-6171?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/update-wordpress-and-plugins.yml [hunk 1]
```diff
@@ -155,10 +155,17 @@ jobs:
...
+      #   git remote set-url origin https://${GH_PAT}@github.com/mailpoet/mailpoet
+      #   git push -f origin HEAD:refs/heads/update-plugins-and-wordpress-test
       - name: Push changes
         if: env.CHANGES_DETECTED == 'true'
         run: |
-          git push origin HEAD:refs/heads/update-plugins-and-wordpress
+          git push -f origin HEAD:refs/heads/update-plugins-and-wordpress
 
       # Create a pull request if there are changes
       - name: Create Pull Request
```
Comment (lysyjan) [inline .github/workflows/update-wordpress-and-plugins.yml:171]: @costasovo Does this work without an error when a PR already exists? I don't know. Just asking.

Comment (costasovo) [reply .github/workflows/update-wordpress-and-plugins.yml:171]: In my testing with the act tool, it worked. You can see history in the testing PR https://github.com/mailpoet/mailpoet/pull/5761 also the PR desription was updated (I added "UPDATED!") string.

Comment (lysyjan) [reply .github/workflows/update-wordpress-and-plugins.yml:171]: Thanks 👍 I'm going to merge it.

**Review by lysyjan — APPROVED** [review]
Good job! Thanks!

**Pull request merged**